src/pages/Ingame/index.js

```diff
@@ -1,11 +1,26 @@
 import React, { useState } from "react";
 import styled from "styled-components";
 import GameItem from "../../components/GameItem";
+import RuleBook from "../../components/RuleBook";
+import Modal from "../../components/Modal";
 
 export default function Game() {
   const [mycard, setMycard] = useState("");
   const [isSubmitted, setIsSubmitted] = useState(false);
 
+  const [isModal, setIsModal] = useState(false);
+  const [isInterim, setIsInterim] = useState(false);
+
+  const openRuleModal = () => {
+    setIsModal(!isModal);
+    setIsInterim(false);
```

Comment:
둘다 통일성 있게 가져가면 좋을 것 같습니다!
```suggestion
    setIsModal(true);
    setIsInterim(false);
```

Reply:
참고하겠습니다!